mixmatch/services.py

```diff
@@ -108,3 +116,13 @@ def _is_reverse(order):
         return True
     else:
         raise ValueError
+
+
+def _remove_details(volumes):
+    """Delete key, value pairs if key is not in keys"""
+    keys = ['id', 'links', 'name']
+    for i in range(len(volumes)):
+        for key, value in volumes[i].items():
```

Comment:
Actually a dictionary comprehension---I haven't seen many cases for those before!